src/game/protinst.cc

```diff
@@ -406,21 +418,17 @@ int obj_examine_func(Object* critter, Object* target, void (*fn)(char* string))
 
                 snprintf(formattedText, sizeof(formattedText), v66.text, hpMessageListItem.text);
             } else {
-                // %s %s
-                v66.num = 521 + v12;
-                if (!message_search(&proto_main_msg_file, &v66)) {
-                    debug_printf("\nError: Can't find msg num!");
-                    exit(1);
-                }
-
                 MessageListItem v63;
+                // 522: He looks: %s.
+                // 523: She looks: %s.
+                // 524: It looks: %s.
                 v63.num = 522 + stat_level(target, STAT_GENDER);
```

Comment:
you can add a check here for `critter_body_type(target) != BODY_TYPE_BIPED` and then set `num` to 524 - "It looks: %s." See example above at line 280 of unmodified file.